lab_6_pipeline/pipeline.py

```diff
@@ -86,22 +155,32 @@ def __init__(self, position: int, text: str, tokens: list[ConlluToken]):
         """
         Initializes ConlluSentence
         """
+        self._position = position
+        self._text = text
+        self._tokens = tokens
 
     def get_conllu_text(self, include_morphological_tags: bool) -> str:
         """
         Creates string representation of the sentence
         """
+        return self._format_tokens(include_morphological_tags)
```

Comment:
this method should create sentence block from conllu (id+sentence+words),self._format_tokens should just prepare words

Reply:
fixed